src/main/java/org/opentripplanner/model/Stop.java

```diff
@@ -42,17 +44,39 @@
 
   private HashSet<BoardingArea> boardingAreas;
 
+  public Stop(
+          FeedScopedId id,
+          String name,
+          String code,
+          String description,
+          WgsCoordinate coordinate,
+          WheelChairBoarding wheelchairBoarding,
+          StopLevel level,
+          String platformCode,
+          Collection<FareZone> fareZones,
+          String url,
+          TimeZone timeZone,
+          TransitMode vehicleType
+  ) {
+    super(id, new NonLocalizedString(name), code, description, coordinate, wheelchairBoarding, level);
```

Comment:
I was thinking of just using a `this(id, new NonLocalizedString(name), code...` here, and int the other classes, rather than duplicating the contents.